hack/.golint_failures

```diff
@@ -349,7 +349,6 @@ staging/src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/apis/cr/v1
 staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions
 staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1
 staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver
-staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server
```

Comment:
thanks for fixing this, while we're in favor of holding this until v1 promotion. or it will be some sort of a blocker.

Reply:
hi @yue9944882

Thank for review my first pull request for k8s。
Thank you very much
Best wishes